### PR TITLE
Use toast notifications for build config editor

### PR DESCRIPTION
**Pull request by spadgett**
https://trello.com/c/6WVJdgjL

### CODE REVIEW

**Comment by spadgett** [comment]
[test]

File: app/scripts/controllers/edit/buildConfig.js [hunk 1]
```diff
@@ -312,16 +335,18 @@ angular.module('openshiftConsole')
             // If we found the item successfully, watch for changes on it
             watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
               if (action === 'MODIFIED') {
-                $scope.alerts["updated/deleted"] = {
+                NotificationsService.addNotification({
+                  id: 'edit-build-config-conflict',
```
Comment (jwforres) [inline app/scripts/controllers/edit/buildConfig.js]: you added ids for these two warnings, did you intend to clean them up in hideErrorNotifications?

**Comment by spadgett** [comment]
@jwforres thanks, updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**